docs/source/release/v5.1.0/mantidworkbench.rst

```diff
@@ -168,5 +168,6 @@ Bugfixes
 - Use Jemalloc for memory allocation on Linux so memory can be released to the system.
 - Fixed a bug where instrument view would not update on wheel zoom.
 - Fixed a bug which caused workbench to crash midway through closing normally when logging level was set to debug.
+- Fixed a bug in the Muon ALC interface baseline fit where corrected data from all fits were plotted at the same time in the corrected data tab. Now only corrected data from the most recent fit is plotted.
```

Comment:
this should be in the muon release notes and not the workbench

Reply:
Ah thank you, amended the original commit